pkg/model/model.go

```diff
@@ -4,7 +4,6 @@ package model
 
 // Resource - Describes a resource (node)
 type Resource struct {
-	Kind           string `json:"kind,omitempty"`
```

Comment:
Do we need to keep Kind here?

Reply:
It doesn't look like we are using it anywhere..